scripts/pi-hole/js/settings.js

```diff
@@ -268,9 +268,26 @@ $(function () {
       columnDefs: [
         { bSortable: false, orderable: false, targets: -1 },
         {
-          targets: [0, 1, 2],
+          targets: [0, 1],
           render: $.fn.dataTable.render.text(),
         },
+        {
+          targets: 2,
+          render: function (data) {
+            // Show "unknown", when host is "*"
+            var str;
+            if (data === "*") {
+              str = "<i>unknown</i>";
+            } else {
+              str =
+                typeof data === "string"
+                  ? data.replace(/</g, "&lt;").replace(/>/g, "&gt;").replace(/"/g, "&quot;")
```

Comment:
Shouldn't we use a library provided escaping function here?

Reply:
I copied this from the source code, but I just find out I used the old version.

Reply:
Now using `utils.escapeHtml()`.